tests/drivers/i2c/i2c_slave_api/Kconfig

```diff
@@ -0,0 +1,17 @@
+mainmenu "I2C Slave API Test"
+
+config ZEPHYR_BASE
+	string
+	option env="ZEPHYR_BASE"
+
+config APPLICATION_BASE
```

Comment:
I just glanced over this PR without paying much attention to the details, but this APPLICATION_BASE config option should be removed. It's my fault, I added it to the documentation by mistake. I'm trying to fix it here:

https://github.com/zephyrproject-rtos/zephyr/pull/5399

Otherwise I have no issues with this PR

Reply:
Will fix !